app/views/external_users/claims/basic_fees/_additional_fee_fields.html.haml

```diff
@@ -23,7 +23,7 @@
     .form-group
       - fee_type_scope = fee.fee_type_code.downcase
       = f.adp_text_field :quantity,
-        label: t(".#{fee_type_scope}.quantity"),
+        label: t('.quantity'),
```

Comment:
+1 - good to see removal of dynamic translations as will enable used on i81n-task missing and unused checks.